packages/platform-browser/src/hydration.ts

```diff
@@ -159,31 +74,36 @@ function provideZoneJsCompatibilityDetector(): Provider[] {
  * export class AppModule {}
  * ```
  *
- * @see {@link withNoDomReuse}
- * @see {@link withNoHttpTransferCache}
+ * `domReuse: false` disables DOM nodes reuse during hydration. Effectively makes
```

Comment:
We can update the docs to introduce the config object:
```suggestion
 * The function accepts a configuration object as the first argument. The configuration object may contain
 * the following fields:
 *  - `domReuse` - a boolean value (`true` by default) that controls whether Angular should match DOM nodes during hydration.
 * - `httpTransferCache` - either a boolean to enable/disable transferring cache for eligible requests performed
 *   using `HttpClient`, or an object, which allows to configure cache parameters, such as which headers should
 *   be included (no headers are included by default).
```
and after that, we can talk about the `domReuse: false` case (like you have it here).